### ISSUE TITLE
HTTP to HTTPS redirection

### ISSUE THREAD

**Issue by joahua** [govau/uikit#44]
https://github.com/govau/uikit/issues/43 requested an SSL cert for http://guides.service.gov.au/design-guide/ which is available under the *.service.gov.au wildcard certificate.

However, we should also implement HTTP to HTTPS redirection for this domain.

**Comment by dominikwilkowski** [comment]
I would like to bump this up in urgency seeing that Firefox and Chrome now label non SSL connections unsafe:

https://www.chromium.org/Home/chromium-security/marking-http-as-non-secure
https://blog.mozilla.org/security/2015/04/30/deprecating-non-secure-http/comment-page-1/

**Comment by joahua** [comment]
https://github.com/AusDTO/dta-website/blob/develop/nginx.conf#L69 is how the DTA site implements this, per the recommended http://docs.cloudfoundry.org/buildpacks/staticfile/#force-https

We seem to be using the same buildpack (well, the same name appears here - https://github.com/AusDTO/gov-au-content-guide/blob/master/bin/cideploy.sh) but somehow seem config free… needs further investigation.

**Comment by bjgordon** [comment]
@joahua you only need to add your own nginx.conf if you need to change the configuration from what's in the static buildpack. In the dta website's case, we haven't changed anything there for http -> https redirection. Sounds like you need to set the `FORCE_HTTPS` environment variable in your cf app...

**Comment by joahua** [comment]
Sounds good. Reading [the staticfile buildpack’s config](https://github.com/cloudfoundry/staticfile-buildpack/blob/master/conf/nginx.conf) that the DTA site’s conf is derived from more closely, it seems this is unblocked even if we don‘t have env-setting abilities.

If we do want to use the env var approach, though, Tessa’s noted @dominikwilkowski and @klepas  have CF access to test this one out (and suggested we do HSTS at the same time… I might raise another ticket) when we have capacity.
